src/coq/logicutils/contexts/envutils.ml

```diff
@@ -10,7 +10,10 @@ open Decl_kinds
 open Constrextern
 open Contextutils
 open Evd
-
+open Names
+open Nameutils
+open Tactics (* fresh_id_in_env *)
```

Comment:
If this is used only for one thing, maybe it makes sense to use the fully qualified name when you call that function? Minor though